articles/cosmos-db/how-to-setup-cmk.md

```diff
@@ -314,7 +314,54 @@ Because a system-assigned managed identity can only be retrieved after the creat
         --assign-identity <identity-resource-id>
         --default-identity "UserAssignedIdentity=<identity-resource-id>"  
     ```
+    
+### To use Continuous backup with user-assigned managed identity
```

Comment:
```suggestion
## Use CMK with continuous backup
```

Reply:
Thomas we have to be explicit about user-assigned identity? No? Otherwise we will again invite CRIs.

Reply:
Yes but you're mentioning it in the line right below.